test/liquid_voting/delegations_test.exs

```diff
@@ -60,6 +60,19 @@ defmodule LiquidVoting.DelegationsTest do
       assert {:error, %Ecto.Changeset{}} = Delegations.create_delegation(context[:invalid_attrs])
     end
 
+    test "create_delegation/1 with same participant as delegator and delegate returns error changeset",
+         context do
+      participant = insert(:participant)
+
+      args = %{
+        delegator_id: participant.id,
+        delegate_id: participant.id,
+        organization_id: Ecto.UUID.generate()
+      }
+
+      assert {:error, %Ecto.Changeset{}} = Delegations.create_delegation(args)
```

Comment:
I wonder if we should assert on `changeset.errors` here as well? I looked at the rest of the file and we don't seem to be doing it but thought I'd raise the question. Checking the error at the GraphQL level may be enough.

Reply:
Yeah, we seem to have a convention of only checking the exact error (rather than just checking for any error) at the absinthe layer only. An issue for the upcoming test discussion?

Reply:
Yep, sounds good!